idb/postgres/internal/encoding/encoding_test.go

```diff
@@ -62,8 +62,31 @@ func TestEncodeSignedTxnWithADSynthetic(t *testing.T) {
 		Bytes:  string(nonutf8b),
 	}
 	stxn.EvalDelta.LocalDeltas[1] = ld
+	stxn.EvalDelta.Logs = []string{nonutf8}
```

Comment:
Could you please move this to `TestSignedTxnWithADEncoding()`? This test is old and doesn't test decoding.